HeadstormChallenge/DB.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE "Person"{
+
+ "RecordID" NUMBER NOT NULL PRIMARY KEY,
+ "Name" VARCHAR(40) NOT NULL,
+ "CPhone" CHAR(12),
+ "WPhone" CHAR(12),
+ "Email" VARCHAR(40),
+ "Address" VARCHAR(40),
+ "BasicWidgetO" NUMBER,
+ "AdvWidgetO" NUMBER,
+ "ProtectPlan" BOOLEAN
+ FOREIGN KEY ("UserID") REFERENCES "Record"("UserID")
```

Comment:
How did you determine that you should use UserID as a foreign key? and Where is the Record table?

Reply:
Hi Faisal,
I determined to use UserID as a foreign key when I originally was going to have two tables and have UserID be the link between the two tables that are the Record and Person table. However, I ended up opting for just one table and should have deleted line 12 as there is no other table. Apologies.